src/wdl/workflows/RegisterEGADatasetFinalizeSubmission/RegisterEGADatasetFinalizeSubmission.wdl

```diff
@@ -0,0 +1,51 @@
+version 1.0
+
+workflow RegisterEGADatasetFinalizeSubmission {
+
+    input {
+        String submission_accession_id
+        String ega_inbox
+        String password
+        String policy_title
+        String library_strategy
+        Array [String] run_accession_ids
+        String? dataset_title
+        String? dataset_description
+    }
+
+}
+
+task RegisterDatasetFinalizeSubmission {
+    input {
+        String submission_accession_id
+        String ega_inbox
+        String password
+        String policy_title
+        String library_strategy
+        Array [String] run_accession_ids
+        String? dataset_title
+        String? dataset_description
+    }
+
+    command {
+        python3 /src/scripts/ega/register_dataset_and_finalize_submission.py \
+            -submission_accession_id ~{submission_accession_id} \
+            -user_name ~{ega_inbox} \
+            -password ~{password} \
+            -policy_title ~{policy_title} \
+            -library_strategy ~{library_strategy} \
+            -run_accession_ids ~{run_accession_ids} \
+            -dataset_title ~{dataset_title} \
+            -dataset_description ~{dataset_description} \
+    }
+
+    runtime {
+        preemptible: 3
+        docker: "schaluvadi/horsefish:submissionV1"
+    }
+
+    output {
+        # TODO what goes here?
```

Comment:
I dont think we necessarily need an output here since no other tasks will be reading from it.